src/wandb_osh/lightning_hooks.py

```diff
@@ -24,11 +24,13 @@ class TriggerWandbSyncLightningCallback(pl.Callback):
     def __init__(
         self,
         communication_dir: PathLike = _comm_default_dir,
+        frequency: int = 1,
```

Comment:
I wonder if there's a better name for this. Because this setting is basically `1/frequency`, right? (the larger this value is, the _less_ often we sync). So perhaps something like `sync_every_n_epochs`?

Reply:
I started with such a name but thought that frequency was shorter and still self-explanatory.
I will make the change to `sync_every_n_epochs`, it is more accurate